src/pages/blog/starlingx-configurable-power-management.md

```diff
@@ -0,0 +1,164 @@
+---
+templateKey: blog-post
+title: New Power Metrics and Power Management capabilities on StarlingX
+author: Davi Frossard
+date: 2023-11-16T11:14:00.000Z
```

Comment:
The StarlingX release is currently scheduled to come out in March 2024. Would it be ok with you to schedule this blog post around the time of the release?

Reply:
Yes, sounds good!